src/main/webapp/PageNotFound/PageNotFound.js

```diff
@@ -0,0 +1,21 @@
+import { Component } from 'react';
+import { hh, div, h1 } from 'react-hyperscript-helpers';
+
+
+export const NotFound = hh(class NotFound extends Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+     return (
+       div({className:"row", style: {"margin":"60px 0"}}, [
+        div({className:"col-xs-12"}, [
+          h1({}, ["Sorry, this page could not be found."])
+         ])
+       ])
+     )
+  }
+})
+export default NotFound;
```

Comment:
This feedback doesn't necessarily need to be applied, but this component can be a [pure functional component](https://logrocket.com/blog/pure-functional-components/) instead. There's no props or state that are used in the render (although those can be used in functional components too...). Something like this should work and be a lot fewer lines of code:
```
export const NotFound = () => {
  return (
    div({className:"row", style: {"margin":"60px 0"}}, [
     div({className:"col-xs-12"}, [
       h1({}, ["Sorry, this page could not be found."])
      ])
    ])
  );
};

```